### ISSUE TITLE
Proper description in the ReadME

### ISSUE THREAD

**Issue by DonaldTsang** [fobo66/vuetaba#11]
It would be ELI5 on "imageboard" and whether it is a chan or booru.
Also it would be good to acknowledge other projects like https://github.com/jubame/aoba

**Comment by fobo66** [comment]
Thanks for the contribution! Both our projects are on the early stages of development, and this project you mentioned is kinda different from what I'm doing. So, I don't think that I need to acknowledge it for now, let's see how it will go, if it won't be abandoned.

Basically, Vuetaba is only a backend for now, and it was inspired by Wakaba. So, it's a chan and not a booru.

**Comment by DonaldTsang** [comment]
@fobo66 thanks for the notice, maybe add a brief description to the ReadME clarifying which "imageboard" it is? That way it will be easier to research on repos like this.
P.S. I love how you are using GraphQL to do this, if you want a feature list of chans I can provide them.

**Comment by fobo66** [comment]
I've updated the README with references
